### ISSUE TITLE
readlink: illegal option -- e | mac OS X

### ISSUE THREAD

**Issue by Rezzakul** [alexppg/helmenv#4]
Thanks for the cool script and the solution for helm version manager.

Unfortunately the script doesn't work in mac os x. But i was able to run it after installing "coreutils" : 
`brew install coreutils`

then replacing `readlink` commands with `greadlink`.

If you like, I can do a pull request and add my script / the script with my modification for mac users. 

Cheers, 
Rezzakul

**Comment by alexppg** [comment]
I'd very grateful if you could do that since I don't have a mac in which I can test your suggestion.

Thanks for taking the time to open both issues!

**Comment by joaquinclearmetal** [comment]
I have the same option with `.zshrc`.
Test on FreeBSD? (maybe some tools similar)

Tested workaround with this in `.zshrc`:
```shell
export PATH="$HOME/.bin:$PATH"
PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
source $HOME/.helmenv/helmenv.sh
```

**Comment by alexppg** [comment]
Fixed in #9. Feel free to reopen if it still doesn't work.
